### PR TITLE
start poc single page app refactor

### PR DESCRIPTION
**Pull request by brendansudol**
* Separates components from PoC page into their own file (right now under `components/v2/`; once I remove all the dead components in the base components dir, I will merge these in there)
* Starts to remove some dead code / components (will continue this in my next PR)

### This pull request is ready to merge when...
- [x] Tests have been updated (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author

### This feature is done when...
- ~Design has approved the experience~
- ~Product has approved the experience~

### CODE REVIEW

**Comment by brendansudol** [comment]
to be reviewed / merged after https://github.com/18F/cms-hitech-apd/pull/411

File: web/src/components/v2/Activities.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        key={a.id}
+        title={`Program Activities › ${activityDisplay(a, i + 1)}`}
+        bgColor="darken-1"
+      >
+        <Collapsible title="Activity Description">
+          <ActivityDescription activity={a} />
+        </Collapsible>
+
+        <Collapsible title="Needs and Objectives">
+          <FormActivityGoals form={`activity-${a.id}-goals`} />
```
Comment (mgwalker) [inline web/src/components/v2/Activities.js]: Eek.  I get it, though.

**Comment by mgwalker** [comment]
Approved from a code perspective.  A quick accessibility check shows a boatload of errors.  Before merging this, I wanted to raise this to product (paging @NAretakis @jeromeleecms @nkkl).

The way the code is structured, it should be pretty quick to get rid of a bunch of the errors since they originate in reused components (so we can change code in one place and it'll be fixed in a dozen places on the site).  I would feel good about merging this now and planning for an a11y spike soon.  I don't think the situation is nearly as dire as the automated tool results might seem to suggest.

**Comment by brendansudol** [comment]
there are no more accessibility errors than there were before this PR, right @mgwalker ?

File: web/src/components/v2/Activities.js (hunk 1 above)
Comment (brendansudol) [reply web/src/components/v2/Activities.js]: this `FormActivityGoals` will be gone entirely soon enough

File: web/files.test.js [hunk 2]
```diff
@@ -1,18 +0,0 @@
...
-import './src/actions';
-
-import './src/components/App';
-import './src/components/Counter';
-import './src/components/Root';
-
-import './src/containers/Home';
-
-import './src/reducers';
-import './src/reducers/counter';
```
[the remark this answers is not in the document]
Comment (brendansudol) [reply web/files.test.js]: exactly

**Comment by NAretakis** [comment]
This makes sense. I can also help coordinate some of our bug squashing with our 508 standards. There’s a win here, where if we document this properly, we’ll simultaneously fix accessibility, and create a compliance artifact in the process.

**Pull request merged**